cogs/info.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) idoneam (2016-2019)
+#
+# This file is part of Canary
+#
+# Canary is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Canary is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Canary. If not, see <https://www.gnu.org/licenses/>.
+
+# discord-py requirements
+import discord
+from discord.ext import commands
+import asyncio
+
+from .utils.paginator import Pages
+import math
+
+
+class Info(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command()
+    async def roles(self, ctx):
+        """Returns list of all roles in server"""
+        roleNames = list(map(lambda role: role.name + "\n", ctx.guild.roles))
+        roleNames.reverse()
+        p = Pages(ctx,
+                  item_list=roleNames,
+                  title="All roles in server",
+                  display_option=(3, 20),
+                  editable_content=False)
+
+        await p.paginate()
```

Comment:
In general I'm averse to one-letter names except for indexing – could you give it a more descriptive name?

Reply:
I was following the convention used in quotes.py [line 270](https://github.com/idoneam/Canary/blob/cb71eeb6132d5ee119005398937278aca008bbf9/cogs/quotes.py#L270)

Reply:
I think we'll need to do this in the task/cleanup branch.